src/io/input.jl

```diff
@@ -69,40 +69,102 @@ function readFlowFrame(filename::String)
+- `type`: String, type of datafile, default FCS
+- `method`: transformation method, default arcsinh, optional
+- `cofactor`: Cofactor for transformation, default 5, optional
+- `reduce`: Selected only columns which are defined by lineage and functional, optional,
+    default: false. If false the check for any none columns to be removed (none columns can appear
+    after concatenating FCS files as well as parameter like: time, event length)
+- `sort`: Sort columns by name to make sure the order when concatinating the dataframes, optional, default: false
+- `transform`: Boolean to indicate if the data will be transformed according to method, default: false
+"""
+function loadData(dataPath, data, nWorkers; panel=Nothing(), 
+                type = "fcs", method = "asinh", cofactor = 5,
+                reduce = false, sort = false, transform = false)
+
+    # Split the data according to the number of worker as temp binary file
+    # md can be a metadata file or a single file_name:
+    # generateIO(dataPath, <filename>, nWorkers, true, 1, true)
+    xRange = generateIO(dataPath, data, nWorkers, true, 1, true)
+
+    R =  Vector{Any}(undef,nWorkers)
+
+    # Load the data by each worker
+    # Without panel file, all columns are loaded:
+    # loadData(idx, "input-$idx.jls")
+    # Columns ca be selected by an array of indicies:
+    # loadData(idx, "input-$idx.jls", [3:6;9:11]) <- this will concatenate ranges into arrays
+    # Please note that all optional arguments are by default "false"
+    if type == "fcs"
+        @time @sync for (idx, pid) in enumerate(workers())
+            @async R[idx] = fetch(@spawnat pid loadFCSData(idx, "input-$idx.jls", panel, method,
+                                cofactor,reduce, sort, transform))
+        end
+    else
+        @info "File Type not yet supported!"
+    end
+
+    return R, xRange
+
+end
+
+"""
+    loadFCSData(idx, fn, panel, method, cofactor, reduce, sort, transform)
 
 Load the data in parallel on each worker. Returns a reference of the loaded Data
 
 # Arguments:
 - `idx`: worker index
 - `fn`: filename
-- `md`: Metadata table
-- `panel`: Panel table with a column for Lineage Markers and one for Functional Markers
+- `panel`: Panel table with a column for Lineage Markers and one for Functional Markers, 
+    or Array::{Int} used as column indicies
 - `method`: transformation method, default arcsinh, optional
 - `cofactor`: Cofactor for transformation, default 5, optional
 - `reduce`: Selected only columns which are defined by lineage and functional, optional,
     default: true. If false the check for any none columns to be removed (none columns can appear
     after concatenating FCS files as well as parameter like: time, event length)
 - `sort`: Sort columns by name to make sure the order when concatinating the dataframes, optional, default: true
+- `transform`: Boolean to indicate if the data will be transformed according to method
```

Comment:
I think there should be a default value for `transform`

Reply:
loadFCS() is an internal function and transform is set in the caller function before. It is only passed through